quarkus/distribution/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.tools.ant.filters.ReplaceTokens
+import publishing.GenerateDigest
+
+plugins {
+    id("distribution")
+    id("signing")
+}
+
+description = "Apache Polaris Binary Distribution"
+
+val adminProject = project(":polaris-quarkus-admin")
+val serverProject = project(":polaris-quarkus-server")
+
+distributions {
+    main {
+        distributionBaseName.set("polaris-bin")
+        contents {
+            // Copy admin distribution contents
+            into("admin") {
+                from(adminProject.layout.buildDirectory.dir("quarkus-app"))
```

Comment:
I do not think it's a good idea to do this kind of reference between projects. I think it is preferable to define a gradle configuration in each "source" project and copy from it here.

Reply:
Fixed in the new commit by using artifact dependency instead of cross-module task dependency